app/assets/v2/js/pages/bounty_details.js

```diff
@@ -345,7 +345,7 @@ pull_interest_list(result['pk'], function(is_interested){
         actions.push(entry);
     }
 
-    if(result['status']=='open' || result['status']=='started' || result['status']=='submitted' ){
+    if(result['status']=='open' || result['status']=='started' || result['status']=='submitted' ){ // TODO -- do we want to change how this functions if expired due to https://github.com/gitcoinco/web/pull/566 ?
```

Comment:
🤔

Reply:
i think the answer is that if theres a mock expiration date we want to display all these options even if the bounty is expired... sound okay to you?